spec/features/cookbook_directory_spec.rb

```diff
@@ -28,4 +28,12 @@
       expect(all('.simple_listing li').size).to eql(5)
     end
   end
+
+  it 'shows link for cookbook adoptions' do
+    within '.learn_about_cookbooks_content' do
+      expect(page).to have_content('How do I adopt a cookbook?')
+      click_link('Click here')
```

Comment:
If you use my suggestion for the change above, this would also need to be changed.

    click_link('list of adoptable cookbooks')